extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerLocalClient.scala

```diff
@@ -45,6 +48,24 @@ class RangerLocalClient extends RangerAdminRESTClient with RangerClientHelper {
       lastKnownVersion: Long): ServicePolicies = {
     policies
   }
+
+  override def getUserStoreIfUpdated(
```

Comment:
`RangerLocalClient` is put in `ranger-spark-security.xml` as implementation of `RangerAdminRESTClient` interface, and it must override `getUserStoreIfUpdated` for Ranger 2.1+ but it will comes to error while overriding nothing for Ranger 2.0 and below.

I have difficulty in shim the version difference. What shall we do here? Make authz support 2.1+ only, or just remove support this use case ?

Reply:
I have removed the ut which needs impleation for `getUserStoreIfUpdated` that would break compilation compatibility for Ranger 2.0 or below. And I will put the removed ut and necessary resources in rule json in this PR as remark, after it ismerged.
@yaooqinn